backend/src/data/migrations/20220918074339_change_interview_status_to_enum.ts

```diff
@@ -0,0 +1,34 @@
+import { Knex } from 'knex';
+
+enum InterviewStatus {
+  PENDING = 'Pending',
```

Comment:
```suggestion
  PENDING = 'pending',
```
all statuses should be lewercased

Reply:
I thought of that, but we already have uppercased values in DB, wouldn't this constraint with lowercase break the DB? Or should I update all the uppercased values from DB to the lowercased ones?